src/main/java/com/charity/charitydonationstracker/entity/CharityDonation.java

```diff
@@ -0,0 +1,33 @@
+package com.charity.charitydonationstracker.entity;
+
+import jakarta.persistence.*;
```

Comment:
Dont use a * for imports. Instead remove the * and include the specific classes you need.

Reply:
Done